charts/argo-cd/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: v2.14.11
 kubeVersion: ">=1.25.0-0"
 description: A Helm chart for Argo CD, a declarative, GitOps continuous delivery tool for Kubernetes.
 name: argo-cd
-version: 7.8.28
+version: 7.8.30
```

Comment:
I don't understand why it went up twice?

Reply:
Maybe due to two occurences in the code?
Is it maybe treatet as two deps?